### PR TITLE
Link to tutorial and  remove checkout step

### PR DESCRIPTION
**Pull request by pchico83**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +24,9 @@ The length of time to wait for completion. Values should contain a corresponding
 
 ### `scope`
 
-**Required**  The scope of the Okteto preview environment to create.
+The scope of the Okteto preview environment to create.
```
Comment (ifbyol) [inline README.md:27]: Isn't it required anymore?

**Review by ifbyol — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pchico83) [reply README.md:27]: AFAIK, it was never required. It defaults to `personal`

Comment (pchico83) [reply README.md:27]: you are right, it is required, but I think it should be optional. I will update this PR.
@ifbyol or do you remember a reason to make it required? in the CLI it is optional

**Review by jLopezbarb — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ifbyol) [reply README.md:27]: @pchico83  No, I don't remember any reason, I think defaulting to personal makes sense. Can you mentioning it on the field description?

**Review by ifbyol — APPROVED** [review]

**Pull request merged**